asab/storage/mongodb.py

```diff
@@ -225,8 +284,8 @@ class MongoDBUpsertor(UpsertorABC):
 	def generate_id(cls):
 		return bson.objectid.ObjectId()
 
-
-	async def execute(self, custom_data: typing.Optional[dict] = None, event_type: typing.Optional[str] = None):
```

Comment:
We cannot change the signature of the existing method ...

Reply:
I am not sure I understand this correctly. Because in my reading the signature does not change. The "session" is not provided by the user of the method (but rather by the transactional decorator), so all code using this "execute" remains exactly the same. But maybe your concern was something else